### DISCUSSION TITLE
Como o Professor Aloprado roubou o Natal

### DISCUSSION THREAD

**Discussion by vish-blog** [vish-blog/comments#391] [Imagens]
Post de referência: https://www.vish.blog.br/imagens/como-o-professor-aloprado-roubou-o-natal/

**Comment by brunoaugusto** [comment]
De onde veio essa foto?

**Comment by najarin-the-loremaster** [comment]
> [Bruno Augusto](</perfil/?user=brunoaugusto>) disse [segunda-feira, 4 de abril de 2022](<https://www.vish.blog.br/imagens/como-o-professor-aloprado-roubou-o-natal/#comment_1087777291>):

> De onde veio essa foto?

A foto é de meados dos anos de 1999/2000 e foi tirada as gravações de "O Grinch" (*How the Grinch Stole Christmas*) e "Professor Aloprado 2" (*Nutty Professor II: The Klumps*)

**Comment by one-eyed-willy** [comment]
> [Najarin, the Loremaster](</perfil/?user=najarin-the-loremaster>) disse [segunda-feira, 4 de abril de 2022](<https://www.vish.blog.br/imagens/como-o-professor-aloprado-roubou-o-natal/#comment_1087781932>):

> A foto é de meados dos anos de 1999/2000 e foi tirada as gravações de "O Grinch" (*How the Grinch Stole Christmas*) e "Professor Aloprado 2" (*Nutty Professor II: The Klumps*)

Tava aqui pensando... Imagina se o Eddie Murphy tivesse sido o Grinch

**Comment by najarin-the-loremaster** [comment]
> [Willy Caolho](</perfil/?user=one-eyed-willy>) disse [segunda-feira, 4 de abril de 2022](<https://www.vish.blog.br/imagens/como-o-professor-aloprado-roubou-o-natal/#comment_1087783700>):

> Tava aqui pensando... Imagina se o Eddie Murphy tivesse sido o Grinch

Ele até foi considerado para o papel antes de optarem pelo Jim Carrey — o que, na minha humilde opinião, foi a melhor decisão que eles puderam tomar para o filme

**Comment by angellisy** [comment]
> [Najarin, the Loremaster](</perfil/?user=najarin-the-loremaster>) disse [segunda-feira, 4 de abril de 2022](<https://www.vish.blog.br/imagens/como-o-professor-aloprado-roubou-o-natal/#comment_1087787170>):

> Ele até foi considerado para o papel antes de optarem pelo Jim Carrey — o que, na minha humilde opinião, foi a melhor decisão que eles puderam tomar para o filme

E eu achando que era só uma foto curiosa que eu achei por aí :o
